packages/kbn-pm/src/commands/reset.ts

```diff
@@ -63,8 +63,10 @@ export const ResetCommand: ICommand = {
       }
     }
 
+    const repoRoot = resolve(__dirname, '../../../..');
```

Comment:
The third argument to `run()` is an object with the `kbn` property, which would allow you to use `kbn.getAbsolute()` which will return the repo root, rather than re-resolving it here. For instance https://github.com/elastic/kibana/blob/0cae95ca74df2ab113db8a5d7f765805529dc68c/packages/kbn-pm/src/commands/bootstrap.ts#L143

Reply:
Thanks, didn't know about that one.